### ISSUE TITLE
ViaBrowser deletes your files ;] unfortunately

### ISSUE THREAD

**Issue by issue1001** [via-browser/Via#435]
When you're uninstalling ViaBrowser, it deletes all files you have downloaded with Via (!) unless you rename/move them to a different directory. I have learnt that the hard way ;] Please fix that. Thanks

(Happens on Android 7.0 smartphone, non-rooted)

**Comment by LakorTi** [comment]
I'm very sorry about this. But Via will save the downloaded file in the `/sdcard/Download` directory. Generally, the file in the download directory will not be affected by the uninstallation of the application. I have just tried to uninstall the application after downloading the file, and the downloaded file still exists. This may be due to other reasons.

**Comment by issue1001** [comment]
Thank you for your answer. I like your browser very much.     I have waited a few versions of Via browser or if it resolves by itself. This quite nasty problem I mentioned, still persists.   For downloads I use a separate directory called &#34;VIA&#34; in my root storage folder - in VIA it says:  Download location:  /storage/emulated/0/VIA   When I download a file or more files with Via browser, and rename a few of them after download, then at uninstall of Via browser, the files renamed are left in directory but those untouched (with original names) are deleted from that directory.  So if I will not change names or move files to other directory, then my all downloads are gone.     
            
          
     
      
       
        Dnia 3 kwietnia 2020 19:35 lakor &lt;notifications@github.com&gt; napisał(a):
       
    
       
         
 I&#39;m very sorry about this. But Via will save the downloaded file in the  /sdcard/Download  directory. Generally, the file in the download directory will not be affected by the uninstallation of the application. I have just tried to uninstall the application after downloading the file, and the downloaded file still exists. This may be due to other reasons.
